about.html

```diff
@@ -0,0 +1,162 @@
+                            <h3> When</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Thursday, Jun 30 2022<br>
+                                9:00 AM - 5:30 PM (EAT)
+                            </p>
+                        </div>
+                    </div>
+                    <div class="when">
+                        <div class="when-header">
+                            <a class="fa fa-map-marker" href="#"></a>
+                            <h3> Where</h3>
+                        </div>
+                        <div class="when-desc">
+                            <p>
+                                Africa's Talking HQ<br>
+                                Mzima Springs Rd, Lavington, Nairobi<br>
+                                Nairobi, 00100<br>
+                            </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="about-desc">
+            <h3>Africa's Talking</h3>
+            <p>
+                powers communications solutions across Africa with simplified 
+                access to telco infrastructure, developers use our powerful SMS, 
+                USSD, Voice, Airtime, and Payments APIs to bring their ideas to life, 
+                as they build and sustain scalable businesses.
+            </p>
+        </div>
+        <section class="past-summit">
+            <div class="summit-header about-desc">
+                <h3>See The past Africa's Talking</h3>
+                <p>
+                    Tale a look at the last two Africa's talking 
+                    which took palce in Nairobi.
+                </p>
+            </div>
+            <div class="summit-images">
+                <div class="summit">
+                    <div class="summit-cover">
+                        <div class="about-desc summit-desc">
+                            <h3>2021</h3>
+                            <p>Africa's Talking 2021 in Nairobi</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="summit summit-2">
+                    <div class="summit-cover">
+                        <div class="about-desc summit-desc">
+                            <h3>2020</h3>
+                            <p>Africa's Talking 2020 in Nairobi</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+        
+    </main>
+    <footer class="about-footer">
+        <div class="quick-links">
+            <h3>QUICK LINKS:</h3>
+            <ul class="q-links">
+                <li><a href="#">EVENTS</a></li>
+                <li><a href="#">CHAPTERS</a></li>
+                <li><a href="#">START BUILDING</a></li>
+                <li><a href="#">JOIN SLACK</a></li>
+
+            </ul>
+        </div>
```

Comment:
- [ ] This section is not present on the original design, kindly consider removing it from the `About` page and changing it to the partner's section on the `Home` page so your design is similar to the CC design